web/app/themes/imb/base.php

```diff
@@ -1,28 +1,24 @@
 <?php get_header(); ?>
-<body <?php body_class(); ?>>
 
   <!--[if lt IE 8]>
 	<div class="alert alert-warning">
 		<?php _e( 'You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.', 'roots' ); ?>
 	</div>
   <![endif]-->
 
-	<?php
-	do_action( 'get_header' );
-	get_header();
-	?>
-
   <div class="wrap container" role="document">
-	<div class="content row">
-	  <main class="main" role="main">
-		<?php require roots_template_path(); ?>
-	  </main><!-- /.main -->
-		<?php if ( roots_display_sidebar() ) : ?>
-		<?php endif; ?>
-	</div><!-- /.content -->
+		<div class="content row">
+			<main class="main" role="main">
+			<?php require roots_template_path(); ?>
+			</main><!-- /.main -->
+			<?php if ( roots_display_sidebar() ) : ?>
+			<?php endif; ?>
+		</div><!-- /.content -->
   </div><!-- /.wrap -->
 
 	<?php get_footer(); ?>
 
+	</div><!-- /.ccfw-banner-background class set in header -->
```

Comment:
Also if this does not need to be wrapping the whole page you could stick the close body and html into the footer.php